src/createServer.js

```diff
@@ -1,9 +1,24 @@
-/* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const url = new URL(req.url.slice(1), `http://${req.headers.host}`);
```

Comment:
The use of `req.url.slice(1)` might cause issues if the URL path is empty, as it removes the leading slash. Consider using `req.url` directly or ensure that the URL is correctly formatted to avoid potential errors.